src/version.h

```diff
@@ -11,7 +11,7 @@
  */
 
 
-static const int PROTOCOL_VERSION = 70210;
+static const int PROTOCOL_VERSION = 70211;
```

Comment:
already bumped, see https://github.com/dashpay/dash/pull/2256

Reply:
Removing this bump can create problems with testing in devnets and testnet, they are using 70210 protocol without this functionality already.